bloom-housing/bloom
# fix: remove password from logs

# Pull Request Template

## Issue Overview

This PR addresses #3727 


## Description

We are currently logging failed requests in the frontend server logs. This has the side effect of logging emails and passwords on failed logins. This PR masks the passwords and emails if they exist.

We can have further discussions on if we should be logging this at all.

Also we will most likely need to go into Doorway and delete all of the logs so that historical logins are removed

## How Can This Be Tested/Reviewed?

Attempt to login with invalid credentials. The log in the terminal should show "****" in place of the sensitive data

## Checklist:


## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.
